care/facility/models/asset.py

```diff
@@ -46,11 +46,45 @@ class RoomType(enum.Enum):
         Facility, on_delete=models.PROTECT, null=False, blank=False
     )
 
+    users = models.ManyToManyField(
+        User,
+        through="AssetLocationDutyStaff",
+        related_name="duty_staff",
+        through_fields=("asset_location", "user"),
+        blank=True,
+    )
+
     middleware_address = models.CharField(
         null=True, blank=True, default=None, max_length=200
     )
 
 
+class AssetLocationDutyStaff(models.Model):
+    asset_location = models.ForeignKey(
+        AssetLocation, on_delete=models.CASCADE, null=False, blank=False
+    )
+    user = models.ForeignKey(User, on_delete=models.CASCADE, null=False, blank=False)
+
+    deleted = models.BooleanField(default=False)
+
+    created_by = models.ForeignKey(
+        User,
+        on_delete=models.PROTECT,
+        null=False,
+        blank=False,
+        related_name="assigned_duty_staff",
```

Comment:
Do we need this relation?

```suggestion
        related_name="+",
```